### DISCUSSION TITLE
Operation Buttons

### DISCUSSION THREAD

**Discussion by Gibbolobbolo** [cnr-isti-vclab/TagLab#43] [Q&A]
Hello guys :) I have downloaded the software on my Mac and it is running smoothly. The projects open up perfectly and all seems to running well. However, the software does not let me use the operation buttons like the "+/- segmentation" and the "4 clicks segmentation". Do you think the problem might be related to the fact I have not downloaded some packages? 

Thanks a lot!

**Comment by maxcorsini** [comment]
Hi, 

Let's try to assign the points by pressing SHIFT + the left mouse button.
